Bullwinkle.class.nut

```diff
@@ -95,10 +99,11 @@ class Bullwinkle {
     // Parameters:
     //      name            The message name
     //      data            Optional data
+    //      ts              Optional timestamp for the data
```

Comment:
I suppose having proper docs would help with that :)

Its just a timestamp associated with the data.  If not passed in, Bullwinkle will use the time that .send() is called (which is the v2.3.1 behavior) but this change allows for any timestamp that a developer wants to use.

Imagine a battery powered device that records data 1/minute and writes it to nv/spiflash.  Then 1/hour it connects to WiFi and send all of the data.  Probably in this case you would just send one big data structure of value/ts but you could send 60 bullwinkle messages with the correct timestamp as part of the message.

The actual reason I went through the trouble of making these changes is for impPager.  With that class, we write data to spiflash when ```bullwinkle.onFail``` happens.  I then read 1 datapoint at a time and send it through bullwinkle (to avoid crashing out of memory if I send all the data at once).  Here, I would like the ability to send the data with the original timestamp the data was created, not the timestamp that I call ```bullwinkle.send()``` for the nth time.